src/frontends/pytorch/src/transforms/fftn_complex_replacer.cpp

```diff
@@ -0,0 +1,151 @@
+            auto full_s_cond = std::make_shared<v1::Equal>(raw_s_input_maybe, const_neg_1);
+            auto full_s_values = std::make_shared<v8::Gather>(input_shape, dim, const_0);
+            s = std::make_shared<v1::Select>(full_s_cond, full_s_values, raw_s_input_maybe);
+        } else {
+            // Value for s was set to default, use full size for all dimensions.
+            s = std::make_shared<v8::Gather>(input_shape, dim, const_0);
+        }
+
+        // Handle norm parameter indicating normalization mode to use. Defaults to "backward".
+        std::string norm;
+        if (const auto& fw_node_mode =
+                std::dynamic_pointer_cast<ov::op::util::FrameworkNode>(fftn_op->input_value(3).get_node_shared_ptr())) {
+            const auto& attrs = fw_node_mode->get_attrs();
+            if (attrs.find("string_value") != attrs.end()) {
+                norm = attrs.at("string_value");
+            } else {
+                norm = "backward";
+            }
+        } else {
+            add_exception_to_fw_node(fftn_op, "aten::fft_fftn: could not retrive value for norm attribute.");
+            return false;
+        }
+
+        auto rdft = std::make_shared<v9::RDFT>(input, dim, s);
+
+        // Apply normalizations
+        auto n_int = std::make_shared<v1::ReduceProd>(s, const_0);
+        auto n = std::make_shared<v1::ConvertLike>(n_int, rdft);
+        std::shared_ptr<ov::Node> normalized_fftn;
+        if (norm == "forward") {
+            // Normalize by 1/n
+            normalized_fftn = std::make_shared<v1::Divide>(rdft, n);
+        } else if (norm == "backward") {
+            // No normalization
+            normalized_fftn = rdft;
+        } else if (norm == "ortho") {
+            // Normalize by 1/sqrt(n)
+            auto sqrt_n = std::make_shared<v0::Sqrt>(n);
+            normalized_fftn = std::make_shared<v1::Divide>(rdft, sqrt_n);
+        } else {
+            add_exception_to_fw_node(
+                fftn_op,
+                "aten::fft_fftn: unrecognized normalization mode. Only forward, backward and ortho are supported.");
+            return false;
+        }
+
+        // Replace outputs that are either torch operators aten::real or aten::imag. Apply squeeze to remove last
+        // dimension used to concatenate.
+        auto fftn_outs = fftn_op->get_users();
+        bool rval = false;
+        for (auto& out : fftn_outs) {
+            if (auto real_op = cast_fw_node(out, "aten::real")) {
+                auto squeezed = std::make_shared<v0::Squeeze>(normalized_fftn->output(0), const_neg_1);
+                copy_runtime_info({fftn_op, real_op}, squeezed);
+                squeezed->set_friendly_name(real_op->get_friendly_name());
+                replace_node(real_op, squeezed);
+                rval = true;
+            }
+        }
+        add_exception_to_fw_node(fftn_op, "aten::fft_fftn: Unsupported output node. Only aten::real is supported.");
```

Comment:
probably you should check that rval == false for adding exception, otherwise you add exception to node that correctly converted